pkg/config/config.go

```diff
@@ -215,7 +215,20 @@ func (vcConfig *VCConfig) validate() error {
 	return nil
 }
 
+func removeEmptyItem(source []string) []string {
+	target := make([]string, 0)
+	for _, value := range source {
+		if len(value) == 0 {
+			continue
+		}
+		target = append(target, value)
+	}
+	return target
```

Comment:
From what I learned from WCPSVC team these days, they prefer the below way, they said the performance is slightly faster.
target := make([]string, 0, len(source))
for i, value := range source {
    if len(value) !=0 {
    target[i] = value
}
}

Reply:
but we don't know the length of target, e.g source length is 4, but 3 of them are empty item. then target only has 1 item.